velox/functions/lib/window/NthValue.cpp

```diff
@@ -117,7 +131,8 @@ class NthValueFunction : public exec::WindowFunction {
       if (offsets_->isNullAt(i)) {
         rowNumbers_[i] = -1;
       } else {
-        vector_size_t offset = offsets_->valueAt(i);
+        vector_size_t offset =
```

Comment:
It seems like we would never encounter this code for Spark invocations, so is this static_cast needed ? Just asking because its repeated on a per-row basis.

Reply:
Yes, spark would not run to this line, I removed this cast, thanks.